imap_processing/tests/lo/test_lo_l1c.py

```diff
@@ -271,3 +313,46 @@ def test_calculate_exposure_times(l1b_de):
         expected_exposure_times,
         atol=1e-2,
     )
+
+
+@pytest.mark.parametrize("species", [FilterType.HYDROGEN, FilterType.OXYGEN])
+def test_set_background_rates(
+    l1b_de_spin, anc_dependencies, attr_mgr, species, expected_bg
+):
+    # Arrange
+    pointing_start_met = 473389100.0
+    pointing_end_met = 473472100.0
+
+    # Act
+    rates, uncert, err = set_background_rates(
+        pointing_start_met, pointing_end_met, species, anc_dependencies, attr_mgr
+    )
+
+    # Assert
+    np.testing.assert_array_equal(
+        rates.values,
+        expected_bg[0],
+    )
+    np.testing.assert_array_equal(
+        uncert.values,
+        expected_bg[1],
+    )
+    np.testing.assert_array_equal(
+        err.values,
+        expected_bg[2],
+    )
+
+
+def test_set_background_rates_species_error(anc_dependencies, attr_mgr):
+    # Arrange
+    pointing_start_met = 473389100.0
+    pointing_end_met = 473472100.0
+    species = FilterType.DOUBLES
+
+    # Act
+    with pytest.raises(
+        ValueError, match=f"Species must be 'h' or 'o', but got {species.value}."
```

Comment:
Just keep it simple for the match and look at the first few characters to generally say it was this error versus from somewhere else, we don't really care about the exact text.
```suggestion
        ValueError, match=f"Species must be 'h' or 'o'"
```